_data/adityarathi1998/adityarathi19.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Aditya Rathi
+institution: Shri G. S. Institute of Technology and Science, Indore
+profile_pic: adityarathi1998.png
```

Comment:
Thanks for the changes - however, the profile picture path does not match the file name (```adityarathi19.png```).

Reply:
Thanks for suggesting the changes, renamed the profile pic file to the picture path.